gap/Magic.gd

```diff
@@ -303,6 +303,17 @@
 #!             <Br/>
 #!             <E>Default value: <C>[ ".", "gap", "lib", "examples", "examples/doc" ]</C>.</E>
 #!         </Item>
+#!         <Mark><A>gap_root_relative_path</A></Mark>
+#!         <Item>
+#!             Either true or a string representing a relative path. &GAPDoc;s MakeGAPDocDoc,
+#!             which is called to build the documentation, supports an optional parameter <A>gaproot</A>
+#!             that makes the links to other documentation entries relative. So this path, if present, should
+#!             be the relative position of the &GAP; root directory to the place where the documentation will be stored.
+#!             The default for this is not having a path, i.e., MakeGAPDocDoc will produce an HTML document with
+#!             absolute paths. If this value is set to true, the relative path <C>../../..</C> is used.<Br/>
+#!             There is also the possibility to set the option <C>relativePath</C> instead of using this value.
+#!             If both the option and the entry are set, the option is used.
+#!         </Item>
 #!
```

Comment:
Hmm, and we might want to *not* document `relativePath` at all, so that we can remove or at least rename it (it sticks out right now, because it uses CamelCase, where most other variables use snake_case)